gdx/src/com/badlogic/gdx/math/MathUtils.java

```diff
@@ -59,8 +59,15 @@ private MathUtils () {
 		static {
 			for (int i = 0; i < SIN_COUNT; i++)
 				table[i] = (float)Math.sin((i + 0.5f) / SIN_COUNT * radFull);
-			for (int i = 0; i < 360; i += 90)
-				table[(int)(i * degToIndex) & SIN_MASK] = (float)Math.sin(i * degreesToRadians);
+			// Calculating the radians from degrees is just a little imprecise; this
+			// was slightly off for 180 degrees (giving -8.742278E-8 instead of 0).
+			// It's easier to understand to assign the exact values like this.
+			// Only these right angles are given the most precise measurement, so
+			// other angles should be compared with a tolerance.
```

Comment:
This comment helps understand the change but doesn't document the code that well and IMO contains unnecessary information. I would just keep why the code exists (to handle common especial cases with right degree angles caused by float precision errors).

Reply:
I'll shorten it up.